casts/rows/src/UIAccountRow.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import { StyleSheet, View } from 'react-native';
+import BigNumber from 'bignumber.js';
+
+import { TouchableOpacity } from '@tonlabs/uikit.controls';
+import { UICurrency } from '@tonlabs/uicast.numbers';
+import { UIImage, UIImageProps } from '@tonlabs/uikit.media';
+import { UILabel, UILabelRoles, UILabelColors } from '@tonlabs/uikit.themes';
+import { UILayoutConstant, UISkeleton } from '@tonlabs/uikit.layout';
+
+import { UIConstant } from './constants';
+
+export type UIAccountRowProps = {
+    testID?: string;
+    icon: UIImageProps['source'];
+    name: string;
+    description?: string;
+    balance: BigNumber;
+    currencySignIcon: any;
```

Comment:
```suggestion
    currencySignProps?: UICurrencySignProps;
```

What do you think about that? That way we can support all the customisations available for UICurrency sign